### DISCUSSION TITLE
JS file for a single component

### DISCUSSION THREAD

**Discussion by SavarJ** [themesberg/flowbite#110] [Q&A]
Hello everyone!

I am only using flowbite's dropdowns. Is there any way to just link that? Like a CDN for that single component specifically? I don't want to load the whole JS file because I won't be using the other components like modal, accordion, etc.

**Comment by stefanovolpatti** [comment]
Hi @SavarJ 

I think you should create a custom function for your dropdown component.
You can also find the original JavaScript file [here](https://github.com/themesberg/flowbite/blob/main/src/components/dropdown.js) for the dropdown.

**Reply by SavarJ** [reply]
Hi @volp99 

I saw that js file before. But I think they use popper under the hood so I was getting some error. How would I be able to combine everything into one? (Sorry, relatively new developer)
